### PR TITLE
[Docs] Update Explanations: manipulator explanation

### PR DESCRIPTION
**Pull request by psobolewskiPhD**
In this PR I try to update the manipulator explanation, under explanations.
I document
- the change in coordinate systems: manipulators use world coordinates now
- the added `radius` and `handle_size` properties to change the size in world coord
- try to update the implementation description

### CODE REVIEW

**Comment by psobolewskiPhD** [comment]
@kevinyamauchi @alisterburt 
Pretty unsure of this one, but thought it would be useful to open a PR so y'all can make suggestions or provide input to get the facts straight.
❤️

File: docs/dev_guides/manipulators.md [hunk 1]
```diff
@@ -4,19 +4,22 @@ Manipulators are UI elements attached to napari layers to aid with positioning a
...
+ stored in the `manipulator.rotation_matrix` property. Additionally, the `manipulator` has `radius` and `handle_size`
+ properties that allow you to change the size of the visual: the overall radius of the manipulator and the size of 
+ the translator and rotator handles, respectively (again in `world` coordinates). 
 
 ## Translators
 Translators are the UI element on the manipulator responsible for translating the manipulator. When the user 
-clicks on a translator and drags it, the manipulator is translated by the drag vector component parallel to the 
-translator direction. 
+clicks on a translator and drags it, the manipulator is translated by the drag vector component along the axis of the  
+translator.
```
Comment (kevinyamauchi) [inline docs/dev_guides/manipulators.md:18]:
```suggestion
selected translator. 
```

**Review by kevinyamauchi — APPROVED** [review]
LGTM! I've made a minor comment below, but please feel free to merge after addressing it.

File: docs/dev_guides/manipulators.md [hunk 2]
```diff
@@ -4,19 +4,22 @@ Manipulators are UI elements attached to napari layers to aid with positioning a
 being displayed on the canvas. Manipulators can translate and rotate along/around specified axes.
 
 ## Coordinate system
-The manipulator coordinate system is defined with an affine transformation relative to the layer data coordinate system.
-The manipulator transformation is defined as a translation and a 3x3 transformation matrix (rotation and scale). The 
-translation is stored in the `manipulator.translation` property and the transformation is stored in the `manipulator.
-rot_mat` property. The rotation is applied before the translation. 
+The manipulator coordinate system is the napari `world` coordinate system (for more information see [the napari
+ documentation](https://napari.org/stable/guides/3D_interactivity.html#coordinate-systems-in-napari)).
+The manipulator transformation is defined as a translation (of the origin of the manipulator) and a 3x3 transformation
```
Comment (GenevieveBuckley) [inline docs/dev_guides/manipulators.md:9]:
```suggestion
The manipulator transformation is an affine transformation; defined as a translation (of the origin of the manipulator) and a 3x3 transformation
```

Comment (GenevieveBuckley) [reply docs/dev_guides/manipulators.md:9]: Is it a good idea to keep "affine" in the text?

**Review by GenevieveBuckley — APPROVED** [review]

File: docs/dev_guides/manipulators.md (hunk 2 above)
Comment (kevinyamauchi) [reply docs/dev_guides/manipulators.md:9]: I think this is more clear. Thanks @GenevieveBuckley !

**Comment by alisterburt** [comment]
thanks @psobolewskiPhD ! any further changes can happen in new PRs of course :)

**Pull request merged**